### ISSUE TITLE
Optionally disable data shuffling between epochs.

### ISSUE THREAD

**Issue by pkpro** [ludwig-ai/ludwig#3624]
**Is your feature request related to a problem? Please describe.**
I have some data that can be quite large, and this data is sequential. I need this data to be chunked for N epochs without being shuffled. So, for each epoch, the next chunk will be used to fine-tune the model.

**Describe the use case**
Evaluation and testing on each chunk of the long sequential text.

**Describe the solution you'd like**
It would be nice to have an option to disable shuffling between epochs like it was proposed in #185

**Comment by arnavgarg1** [comment]
Hey @pkpro, thanks for raising this request. 

We actually already have an option to disable data shuffling in Ludwig, but we enable data shuffling by default because it's generally considered good practice. In your case, it indeed does make sense to disable it and so it should be easy to do so.

This parameter can be configured using the trainer section of the Ludwig config: https://ludwig.ai/latest/configuration/trainer/

```yaml
trainer:
    should_shuffle: false
```

Let me know if this helps!

**Comment by pkpro** [comment]
Thank you @arnavgarg1, I have restarted the training now, it will take like 12 hours until first epoch is through and I need at least two of them to see the difference. However I have no clue how to check if the data were shuffled or not. Indirectly I'm expecting decrease in perplexity for each next chunk. Any suggestions here? Is there a debug log available?

**Comment by pkpro** [comment]
Thank you @arnavgarg1 once again. I'm closing this ticket, should_shuffle: false is working, and I've got decrease in perplexity as well in error rates.
